src/parsing.jl

```diff
@@ -100,8 +100,10 @@ is not in UTF-8.
 """
```

Comment:
I see further up in this file is `num_bytes(pqv::PQValue)` which has the following in it's docstring:
```LibPQ.jl currently always uses text format, so this is equivalent to C's `strlen`.```

Don't know if the comment needs to change, or if we should have a separate `num_bytes` for binary format. Just though I'd point it out.

Reply:
Yeah just the comment needs to change; that method can be used as is with binary format.

Reply:
@samuel-massinon this comment still needs to change